cloudnet/src/main/java/de/dytanic/cloudnet/template/install/ServiceVersionProvider.java

```diff
@@ -67,18 +57,28 @@ private boolean loadVersionsFromInputStream(InputStream inputStream) {
 
         JsonDocument document = new JsonDocument().read(inputStream);
 
-        if (document.contains("versions")) {
-            Collection<ServiceVersionType> versions = document.get("versions", TypeToken.getParameterized(Collection.class, ServiceVersionType.class).getType());
+        int fileVersion = document.getInt("fileVersion", -1);
+
+        if (VERSIONS_FILE_VERSION == fileVersion) {
+            if (document.contains("versions")) {
```

Comment:
Codacy found an issue: [These nested if statements could be combined](https://app.codacy.com/gh/CloudNetService/CloudNet-v3/pullRequest?prid=6226234)